pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -185,7 +186,11 @@ func (c *Cluster) createSelfSignedCert() (bool, error) {
 
 	// retry a few times in the case that the mgr module is not ready to accept commands
 	for i := 0; i < 5; i++ {
-		_, err := client.ExecuteCephCommand(c.context, c.Namespace, args)
+		_, err := client.NewCephCommand(c.context, c.Namespace, args).RunWithTimeout()
```

Comment:
Should the timeout be an arg? Different commands could have different expected timeouts, right?